packages/flex-plugin-utils-http/src/http.ts

```diff
@@ -99,18 +100,30 @@ export default class Http {
     const pkg = require('../package.json');
     packages[pkg.name] = pkg.version;
 
-    const userAgent = [];
+    const userAgent: string[] = [];
     if (env.isNode()) {
       userAgent.push(`Node.js/${process.version.slice(1)}`, `(${process.platform}; ${process.arch})`);
+      const shell = process.env.SHELL?.split('/')[process.env.SHELL?.split('/').length - 1] || 'unknown';
+      userAgent.push(`shell/${shell}`);
     } else {
       userAgent.push(window.navigator.userAgent);
     }
     if (config.caller) {
       userAgent.push(`caller/${config.caller}`);
     }
-    Object.entries(packages).forEach(([key, value]) => userAgent.push(`${key}/${value}`));
 
-    return userAgent.join(' ');
+    Object.entries(packages).forEach(([key, value]) => userAgent.push(`${key}/${value}`));
+    userAgent.push(`is_ci/${env.isCI()}`);
+
+    return userAgent
+      .filter((element) => {
+        return (
+          !element.includes('flex-plugins-api-utils') &&
```

Comment:
Not needed - you can remove this filter - it's okay if others provide extra bit. Just `return userAgent`